test/runtests.jl

```diff
@@ -1,31 +1,63 @@
 using Oscar
 using Test
+using Distributed
 
 import Random
 
+numprocs_str = get(ENV, "NUMPROCS", "1")
+
+if !isempty(ARGS)
+  jargs = [arg for arg in ARGS if startswith(arg, "-j")]
+  if !isempty(jargs)
+    numprocs_str = split(jargs[end], "-j")[end]
+  end
+end
+
+const numprocs = parse(Int, numprocs_str)
+
+if !isempty(ARGS)
+  jargs = [arg for arg in ARGS if startswith(arg, "-j")]
+  if !isempty(jargs)
+    numprocs = parse(Int,split(jargs[end], "-j")[end])
+  end
+elseif haskey(ENV, "NUMPROCS")
+  numprocs = parse(Int,ENV["NUMPROCS"])
+else
+  numprocs = 1
+end
```

Comment:
This is redundant now and must be removed (actually I don't understand how this even passes CI, as we have `const numprocs = ..` a few lines above?!?)
```suggestion
```

Reply:
I just tried this in repl, and apparently redefining a constant is only a warning, not an error..

`WARNING: redefinition of constant numprocs. This may fail, cause incorrect answers, or produce other errors.`

Reply:
Some further reading reveals that right now in Julia, `const` only fixes the type of the variable and errors if you assign it a value which would change its type. If you assign it a different value of the same type, its undefined behavior, which results in a warning and continues.

```
julia> const a = 5
5

julia> a = 7
WARNING: redefinition of constant a. This may fail, cause incorrect answers, or produce other errors.
7

julia> a = 7.0
ERROR: invalid redefinition of constant a
Stacktrace:
 [1] top-level scope
   @ REPL[3]:1

```